darwin/config.py

```diff
@@ -119,6 +119,6 @@ def get_default_team(self, raise_on_invalid_team: bool = True):
                 return None
         return self.get_team(teams[0])
 
-    def get_all_teams(self):
+    def get_all_teams(self) -> List[Optional[Dict[str, Any]]]:
```

Comment:
I'm wondering: what do we do with `None` teams?

Is there a chance we could filter `None` teams out of the returned list?

Reply:
FIxed, I think :D